source/services/helper.js

```diff
@@ -169,7 +169,7 @@
 
                 // cloneEnabled and cross-tree so copy and do not remove from source
                 if (this.isClone() && this.isForeign()) {
-                  this.parent.insertNode(this.index, angular.copy(nodeData));
+                  this.parent.insertNode(this.index, this.sourceInfo.cloneModel);
```

Comment:
No..we check isClone, not cloneEnabled, right? Isn't it rather the comment that is wrong?
I'd say that is another commit.